eclair-core/src/main/scala/fr/acinq/eclair/api/Service.scala

```diff
@@ -99,10 +99,16 @@ trait Service extends Logging {
                   (paymentHandler ? ReceivePayment(MilliSatoshi(amountMsat.toLong), description)).mapTo[PaymentRequest].map(PaymentRequest.write)
                 case JsonRPCBody(_, _, "send", JInt(amountMsat) :: JString(paymentHash) :: JString(nodeId) :: Nil) =>
                   (paymentInitiator ? SendPayment(amountMsat.toLong, paymentHash, PublicKey(nodeId))).mapTo[PaymentResult]
-                case JsonRPCBody(_, _, "send", JString(paymentRequest) :: Nil) =>
+                case JsonRPCBody(_, _, "send", JString(paymentRequest) :: rest) =>
```

Comment:
You just forgot to update the `help` method (in Service.scala) and the doc (in README.md)